Решение: изменение порядка вывода новостей в консоль.

Kirchik opened the pull request.

Kirchik committed:
Update lab1.cpp

Kirchik committed:
Поправил комментарии

Kirchik committed:
Более компактная запись начальных значений

g4rfan commented on lection-3/lab1.cpp:
```diff
@@ -1,54 +1,86 @@
+struct News	// задаю структуру данных для удобной работы с начальными значениями
+{
+	char* news;
+	double lats;
```
почему множественное число, тут много значений содержится?

Kirchik replied:
Не совсем понял вопрос. Я изменил тип данных float на double потому что при вычисления переменной distance происходит усечение из double к float, поэтому программа запускается некорректно. Или я использовал неправильные названия переменных?

Kirchik committed:
Исправляю недочеты

В 54, 63, 65 строках изменил название переменной.
Вынес переменные temporaryValue и temporaryStr за цикл. 
Добавил переменные temporaryLats и temporaryLons и поменял координаты местами для целостности структуры данных.

Kirchik committed:
Изменил сортировку расстояний.

Убрал ненужные комментарии.
Сделал сортировку расстояний через промежуточную структуру. Таким образом невозможен баг, где меняются местами только значения некоторых методов в структуре.
